CHANGELOG.md

```diff
@@ -2,6 +2,44 @@
 
 All notable changes to this project will be documented in this file. For future plans, see our [Roadmap](https://github.com/precice/precice/wiki/Roadmap).
 
+## develop
+- The python module for the preCICE bindings `PySolverInterface` is renamed to `precice`. This change does not break old code. Please refer to [`precice/src/precice/bindings/python/README.md`](https://github.com/precice/precice/blob/develop/src/precice/bindings/python/README.md) for more informaiton.
+- Use boost stacktrace for cross platform stacktrace printing. This requires Boost 1.65.1
+- Reimplemented the internals of the nearest-projection mapping to signifantly reduce its initialization time.
+- The EventTimings now do a time normalization among all ranks, i.e., the the first event is considered to happen at t=0, all other events are adapted thereto.
+- The old CSV format of the EventTimings log files, split among two files was replaced by a single file, structured JSON format.
+- Fix memory leaks in the XML Attributes and Verificators
+- Add manpages for binprecice and testprecice
+- CMake overhaul:
+  - Convert to target-based system: precice, testprecice, binprecice
+  - New Options:
+    - `PRECICE_Packages` (default ON) to configure CPack
+    - `PRECICE_InstallTest` (default ON) to configure installation of tests.  
+      This includes the binary `testprecice` and necessary files.
+      Use `PREFIX/share/precice` as `PRECICE_ROOT`.
+  - Move CMake files from `tools/cmake-modules` to `cmake/` (general scripts) and `cmake/modules` (find modules).
+  - Migration from file-globbing to explicit source/interface/test-file lists.  
+    Use `tools/updateSourceFiles.py` from project-root to update all necessary files.
+  - `install` target installs:
+     - the library `PREFIX/lib`.
+     - the binaries `PREFIX/bin` and their manfiles `PREFIX/share/man/man1`.
+     - the cmake configuration files `PREFIX/lib/cmake/precice`.
+     - the necessary files to run testprecice `PREFIX/share/precice`. Use this as `PRECICE_ROOT` on installed system.
+  - CTest definition of tests run in isolated working directories:
+    - `precice.Base` for the base test suite
+    - `precice.MPI2` run on 2 MPI ranks
+    - `precice.MPI4` run on 4 MPI ranks
```

Comment:
@fsimonis This is outdated, right?

Reply:
Yes the info regarding the tests is outdated